source/wp-content/themes/wporg-developer-2023/js/user-notes-preview.js

```diff
@@ -3,37 +3,34 @@
  *
  */
 
-( function( $ ) {
-
-	var textarea, tabContentHeight, text, preview, previewContent, tabs, processing, spinner;
+( function () {
+	let textarea, preview, previewContent, tabs, processing, spinner;
 
 	function init() {
-
-		if ( undefined === wporg_note_preview ) {
+		if ( typeof wporg_note_preview === 'undefined' ) {
 			return;
 		}
 
-		textarea = $( '.comment-form textarea' );
-		preview = $( '#comment-preview' );
-		tabs = $( '#commentform .tablist' ).find( 'a' );
-		spinner = $( '<span class="spinner" style="display:none;"></span>' );
-		text = '';
+		textarea = document.querySelector( '.comment-form textarea' );
+		preview = document.querySelector( '#comment-preview' );
+		tabs = document.querySelectorAll( '#commentform .tablist a' );
+		spinner = document.createElement( 'span' );
+		spinner.className = 'spinner';
+		spinner.style.display = 'none';
```

Comment:
This does create the spinner correctly, but apparently it was never styled when we moved to the block theme, so nothing is visible. Not a problem for this PR though.

Reply:
Where can I check the old styles?

Reply:
I wouldn't copy over the old styles, but you could borrow [the loading animation from this CSS](https://github.com/WordPress/wporg-mu-plugins/blob/trunk/mu-plugins/blocks/screenshot-preview-block/src/style.scss#L51) (it's the loading state for images such as [the pattern & style variation screenshots on Themes](https://wordpress.org/themes/twentytwentyfour/)).

Reply:
The spinner didn't have any styled, an empty span was appearing. I added the style, but the time of the spinner is quite small.

https://github.com/user-attachments/assets/43312032-1cfb-42fa-ae86-efd20953b65b